freqtrade/commands/arguments.py

```diff
@@ -116,8 +116,14 @@
 
 NO_CONF_ALLOWED = ["create-userdir", "list-exchanges", "new-strategy"]
 
-ARGS_STRATEGY_UTILS = ["strategy_list", "strategy_path", "recursive_strategy_search"]
+ARGS_STRATEGY_UPDATER = ARGS_COMMON_OPTIMIZE + ["strategy_list"]
```

Comment:
you might want to revert the content of the variable (not necessarily the variable name change, which is good).

args_common_optimize has way too many arguments which are useless for the strategy-updater.